pkg/cmd/services.go

```diff
@@ -69,6 +69,14 @@ func (sc *ServicesCmd) ListServicesCmd() *cobra.Command {
 			return nil
 		},
 	}
+
+	type params struct {
```

Comment:
I believe there is already a type defined for this in that file. It is called ```instanceCreateParams```

Reply:
@maleck13 it is, but we needed to make this change `Properties map[string]map[string]interface{}` otherwise it would not unmarshal

Reply:
@maleck13 done